pyEDAA/ProjectModel/Xilinx/Vivado.py

```diff
@@ -149,7 +149,8 @@ def _ParseFile(self, fileNode, fileset):
 		else:
 			self._ParseDefaultFile(fileNode, filePath, fileset)
 
-	def _ParseVHDLFile(self, fileNode, path, fileset):
+	@staticmethod
```

Comment:
While it's possible to change it to `staticmethod` I'm not a fan of it for this usecase.

With `staticmethod` these internal helper function get public on the class and can be called from outside standalone, which makes no sense and has other risks.